src/galileo/decorator.py

```diff
@@ -843,8 +862,13 @@ def init(
             project: The project name. Defaults to None.
             log_stream: The log stream name. Defaults to None.
             experiment_id: The experiment id. Defaults to None.
+            local_metrics: Local metrics configs to run on the traces/spans before submitting them for ingestion.  Defaults to None.
         """
         GalileoLoggerSingleton().reset(project=project, log_stream=log_stream, experiment_id=experiment_id)
+        GalileoLoggerSingleton().get(
```

Comment:
@dmcwhorter i'm not sure that I understand why we do this line here, to me seems redundant 🤔

Reply:
because when `get` is actually called, we don't have `local_metrics` to set the metrics for the context.

Reply:
true,